images/access_setup/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
```

Comment:
Why did all of this moved to a container image?

Reply:
1. So that you don't have to clone the repo to use it.
2. In the future we may have a controller that is able to create the cluster itself, get the appropriate kubeconfig and automate that action. It's best if it relies on an image.